portfolio/src/main/webapp/script.js

```diff
@@ -37,3 +37,50 @@ window.addEventListener("scroll", function() {
   }
 });
 
+async function getComments() {
+  console.log("in get comments");
+  const numComments = document.getElementById("numComments").value;
+  const response = await fetch('/data?numComments='+ numComments);
+  const comments = await response.json();
+  
+  const commentSection = document.getElementById('comment-list');
+  commentSection.innerText = ""; // clear old comments 
+ 
+  console.log(comments);
+  console.log(numComments);
+  
+  comments.forEach((comment) => {
+      commentSection.appendChild(createCommentElement(comment));
+  });
+}
+
+function createCommentElement(comment){
```

Comment:
See if you can figure out how to add a delete button to each comment. You can probably add a button to that will call your "/delete-comment" action with the id of the current comment list item. Try playing around in the chrome debug tools and see what it takes to add a button to a list item. Or you could even add new `<div>` tags instead!